### PR TITLE
🔥 remove yuidoc

### LINKED ISSUE #511: Remove yuidoc
We are not using yuidoc and it adds a lot of unnecessary boilerplate onto the files.

### PR DESCRIPTION
**Pull request by Slamper**
Closes #511

### CODE REVIEW

**Comment by Slamper** [comment]
For me the tests failed and it refused to push without the second commit.

**Comment by willclarktech** [comment]
@Slamper Which version of node are you using?

**Comment by Slamper** [comment]
@willclarktech v9.2.1

**Comment by willclarktech** [comment]
@Slamper Please use node v6. Those tests will break in later versions of node due to a difference in how `Buffer.from` treats invalid hex strings.

Sorry, this will be properly documented once we fully apply the standards set in https://github.com/LiskHQ/lisk-template

**Comment by Slamper** [comment]
@willclarktech Ok thank you for the information.
I simply removed the test commit 👍

File: .gitignore [hunk 1]
```diff
@@ -49,9 +49,6 @@ jspm_packages
 # IDE directories
 .idea
 
-# Docs
-yuidoc
```
Comment (willclarktech) [inline .gitignore]: 👌 Thanks for being thorough!

File: src/api/liskApi.js [hunk 2]
```diff
@@ -12,43 +12,11 @@
...
- *         testnet: true,
- *         port: '7000',
- *         bannedNodes: [],
- *         nodes: [],
- *         nethash: ''
- *     };
- *
- *
- *     var LSK = lisk.api(options);
-*/
```
Comment (willclarktech) [inline src/api/liskApi.js]: This isn't actually a yuidoc comment, but I'm happy with it being removed as we will have documentation on the documentation site. What do you think @Tosch110 ?

Comment (Tosch110) [reply src/api/liskApi.js]: Yes, agree! It is actually there for yuidoc, it would show it as an example also in yuidoc.

**Comment by willclarktech** [comment]
Thanks @Slamper!

**Pull request merged**